com.unity.netcode.gameobjects/Tests/Editor/NetworkManagerConfigurationTests.cs

```diff
@@ -294,6 +294,36 @@ public void WhenModifyingPrefabListUsingPrefabsAPI_ModificationIsLocal()
             }
         }
 
+        [Test]
+        public void WhenThereAreUninitializedElementsInPrefabsList_NoErrors()
+        {
+            // Setup
+            var networkManagerObject = new GameObject();
+            var networkManager = networkManagerObject.AddComponent<NetworkManager>();
+            networkManager.NetworkConfig = new NetworkConfig
+            {
+                NetworkTransport = networkManager.gameObject.AddComponent<UnityTransport>()
+            };
+
+            try
+            {
+                networkManager.NetworkConfig.Prefabs.NetworkPrefabsLists = new List<NetworkPrefabsList> { null };
```

Comment:
I think rather than starting and stopping the `NetworkManager`, we could instead follow the pattern in the `WhenNetworkConfigContainsOldPrefabList_TheyMigrateProperlyToTheNewList` and simply call `networkConfig.InitializePrefabs()` directly and test the output.